packages/scene-composer/src/components/three-fiber/AnimationComponent/AnimationComponent.tsx

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line import/default
+import React, { Fragment, useEffect } from 'react';
+import * as THREE from 'three';
+import { useFrame } from '@react-three/fiber';
+
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+import { ISceneNodeInternal, useEditorState, useViewOptionState, IAnimationComponentInternal } from '../../../store';
+import { IAnimationComponent } from '../../../interfaces';
+
+const toggleIsAnimationPaused = (
+  object: THREE.Object3D,
+  AnimationMixer: THREE.AnimationMixer,
+  isPaused: boolean,
+  AnimationList: string[],
+) => {
+  try {
+    AnimationList.forEach(function (clip) {
+      const animation = THREE.AnimationClip.findByName(object.animations, clip);
+      AnimationMixer.clipAction(animation).paused = isPaused;
+    });
+  } catch (error) {
+    // console.error(error);
+  }
+};
+
+const list: any[] = [];
```

Comment:
I'm worried that because this variable is in the file but not scoped to a component instance that it may get strange scope changes once we you have multiple components. It's set inside the individual component instance but then the exported getClip function isn't a member variable for the component and it gets called in other files.